openshift/tests-extension/test/olmv1-incompatible.go

```diff
@@ -37,14 +37,26 @@ import (
 )
 
 var _ = Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 operator installation", func() {
-	BeforeEach(func() {
-		helpers.RequireOLMv1CapabilityOnOpenshift()
-	})
 	unique := rand.String(8)
 	nsName := "install-test-ns-" + unique
 	ccName := "install-test-cc-" + unique
 	rbName := "install-test-rb-" + unique
 	opName := "install-test-op-" + unique
```

Comment:
@tmshort ^ It is not in the before each that can be an issue

Reply:
There is only one test in this Describe (and there should only ever be one test). So it does does not matter.

Reply:
Let's play safe :-) 
If it does not matter, let's keep inside the BeforeEach :-)